.travis.yml

```diff
@@ -0,0 +1,7 @@
+language: node_js
```

Comment:
Could we add `sudo: false` and cache the node_modules to give a speed up on builds.